src/components/NewRuntimeModal.js

```diff
@@ -46,7 +51,7 @@ const safeImageDocumentation = 'https://support.terra.bio/hc/en-us/articles/3600
 // distilled from https://github.com/docker/distribution/blob/95daa793b83a21656fe6c13e6d5cf1c3999108c7/reference/regexp.go
 const imageValidationRegexp = /^[A-Za-z0-9]+[\w./-]+(?::\w[\w.-]+)?(?:@[\w+.-]+:[A-Fa-f0-9]{32,})?$/
 
-const MachineSelector = ({ value, machineTypeOptions, onChange }) => {
+const WorkerSelector = ({ value, machineTypeOptions, onChange }) => {
```

Comment:
`MachineSelector` used to be used in two different places to select GCE VMs' as well as Dataproc main machines' machine type. Now that GCE VMs can have GPU(s) attached, this component didn't factor well since we want available GPU type and number to dynamically depend directly on CPU selection, which doesn't apply to Dataproc backend. Therefore, I renamed this component as (Dataproc) `WorkerSelector` which reflects its only purpose going forward. For the rest, I included the `Select` components flatly in `renderCloudComputeProfileSection()` (formerly `renderRuntimeSection`).